modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/constants/CommercePortletKeys.java

```diff
@@ -62,6 +62,10 @@ public class CommercePortletKeys {
 		"com_liferay_commerce_order_web_internal_portlet_" +
 			"CommerceOrderTypePortlet";
 
+	public static final String COMMERCE_PAYMENTS =
+		"com_liferay_commerce_payment_web_internal_portlet_" +
+			"CommercePaymentsPortlet";
```

Comment:
All other portlets use singular names, so this one should be `CommercePaymentPortlet` not `CommercePaymentsPortlet`.

Reply:
Also there are permissions definition based on com_liferay_commerce_payment_web_internal_portlet_CommercePaymentPortlet
we should stick to that name

Reply:
@CrescenzoRegaLF